apps/discoverbos/widget/Project/SubmitButton.jsx

```diff
@@ -0,0 +1,66 @@
+const ownerId = "discover.near";
+const validateForm = props.validateForm;
+const data = props.data;
+
+return (
+    <Widget
+    src={`${ownerId}/widget/Buttons.Green`}
+    props={{
+      disabled: !validateForm(),
+      onClick: () => {
+        if (!validateForm()) return;
+
+        const did = UUID.generate("xxxxxxx");
+        Social.set({
+          thing: {
+            [did]: {
+              "": JSON.stringify(data),
+              metadata: {
+                name: data.name,
+                description: state.description,
```

Comment:
state.description?
Shoudn't it be data.description?

Reply:
yes, correct. thanks